python/test/unit/mesh/test_mesh_partitioners.py

```diff
@@ -20,13 +20,37 @@
 assert (tempdir)
 
 
-@pytest.mark.parametrize("partitioner", [partition_cells_graph])
+partitioners = [dolfinx.cpp.graph.partitioner()]
+try:
+    from dolfinx.cpp.graph import partitioner
+    partitioners.append(partitioner())
+except ImportError:
+    raise
+try:
+    from dolfinx.cpp.graph import partitioner_scotch
+    partitioners.append(partitioner_scotch())
+except ImportError:
+    raise
```

Comment:
Should be pass? We can build just with e.g. Parmetis or KaHiP now?

Reply:
Not yet.